raster/src/main/scala/geotrellis/raster/io/geotiff/BitGeoTiffSegment.scala

```diff
@@ -13,8 +13,8 @@ class BitGeoTiffSegment(val bytes: Array[Byte], cols: Int, rows: Int) extends Ge
     bytesWidth * 8
   }
 
-  def getInt(i: Int): Int = b2i(get(i))
-  def getDouble(i: Int): Double = b2d(get(i))
+  def getInt(i: Int): Int = get(i).toInt
+  def getDouble(i: Int): Double = get(i).toDouble
```

Comment:
Why not use the macro here? Shouldn't it always be better to inline this promotion?